docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+- **TLS certificates:** the `organizations/ordererOrganizations/ordererOrg1.example.com/ordering-service-nodes/osn1.ordererOrg1.example.com/tls` folder contains the TLS certificate (`tls-cert.pem`) and private key (`tls-key.pem`) for the ordering service `osn1` node as well as the TLS CA root cert `tls-ca-cert.pem`. You will need to copy those files into this folder or point to where they exist on your file system.   
+- **Admin client certificates** - `admin-client/` folder contains the TLS certificates for the admin client that will be issuing the `osadmin` commands. The connection between the admin client that calls the `osnadmin` CLI and an orderer requires mutual TLS, although it is not required for the network itself. This means the admin client needs to enroll and register with a TLS CA to generate the TLS certificate (`client-tls-cert.pem`) and private key (`client-tls-key.pem`) that is provided to the `osnadmin` CLI. You will need to copy those certificates and the client TLS CA root cert (`client-tls-ca-cert.pem`) into this folder or point to where they exist on your file system. For simplicity of the tutorial, you could use the same TLS CA that the orderer organization uses. On a production network, they would likely be separate TLS CAs.  
+
+Certificate names used in this example are for illustration purposes and may not reflect the actual names of the certificates generated by the CA. When you generate the certificates you can rename them accordingly to make it easier to differentiate them.
+
+**Important:** You need to create the `config.yaml` file in the orderer organization MSP and orderer local MSP. This file enables "Node OU" support for the channel, a feature that allows the channel to recognize certificates that contain the "admin" OU as representing organization admins (who are admins of the ordering nodes owned by the organization by default). Learn more in the [Fabric CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html#nodeous) documentation.
+
+If you are using a containerized solution for running your network (which for obvious reasons is a popular choice), **it is a best practice to mount these folders (also known as "volumes") external to the container where the node itself is running. This will allow the certificates to be used to create a new node should the node container go down, become corrupted, or be restarted.**
+
+## Prerequisites
+
+Because `osnadmin` commands are performed against an ordering service node (which executes the actions), the ordering service needs to exist before you can create a channel. You can attempt this tutorial with your existing ordering service or deploy a new ordering service. If you decide to use `osnadmin` commands with an existing ordering service, the system channel must first be removed from each ordering node before you can create any new channels. Choose whether you want to use your existing ordering service or deploy a new one:
```

Comment:
"the ordering service needs to exist before you can create a channel"

I see what you mean, but the phrasing here seems misleading. I think of an ordering service as something that is already consenting on channels (either system channel or application channels). While the focus of this tutorial is spinning up OSNs without any channels and then creating/joining an application channel. It seems like once the tutorial is _done_ and you've got three OSNs consenting on a channel, you'll have an ordering service at the end. If you agree with this comment, subsequent references to "ordering service" would also need to be updated.

Reply:
Orderer, ordering node, ordering service, ordering service node are all terms that have been used more or less interchangeably. I am partial "ordering node" (which I think was a Gari suggestion) and don't care much for "ordering service node". Fwiw, there's also "consenter", but that seems to be a term we've strictly used just for nodes that are consenting on an active channel.

I'm actually sort of fine with the grab bag of terms. We've been using all of them for long enough that our users should be used to it. But if we want to tighten up our references and more strictly define these terms going forward, we can. We will just need broader agreement, which will necessitate a discussion.

Reply:
I went through and carefully assessed each occurrence of ordering service and edited it to orderer if it was in the context of bringing up OSNs i.e. before all three are up.

Reply:
Good, I'm fine using "orderer" and "ordering node" and "ordering service node" somewhat interchangeably, but I think you've addressed my concern that we really don't have an "ordering service" until there is a quorum of orderers consenting together.